app/views/shared/_sidebar.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="col container-height sidebar-border">
+  <div class="container-md">
+    <div class="row p-3">
+      <%= simple_form_for :metric, url: review_turnaround_project_metrics_path,
+        method: 'GET', html: { class: 'sidebar-form' } do |f| %>
+          <%= f.input :period,
+            input_html: {class: 'period-selection'},
+            required: false,
+            collection: [[],['Daily', 'daily'], ['Weekly', 'weekly']],
+            selected: value_from_query_param
+          %>
+      <% end %>
```

Comment:
correct me if I'm wrong but I don't think this belongs to the shared sidebar that'll be accessible in all of the views but only in some pages. I'm right then you might need to use something like
```suggestion
      <%= yield :custom_filters %>
```

and then in a view for `review turnaround`
```erb
  <% content_for :custom_filters do %>
    <%= simple_form_for :metric, url: review_turnaround_project_metrics_path,
        method: 'GET', html: { class: 'sidebar-form' } do |f| %>
          <%= f.input :period,
            input_html: {class: 'period-selection'},
            required: false,
            collection: [[],['Daily', 'daily'], ['Weekly', 'weekly']],
            selected: value_from_query_param
          %>
      <% end %>
  <% end %>
```

Reply:
actually even the filter is going to be refactored, this was the initial idea to show the weekly metric but, my next card is to build the UI as we talked yesterday